modules/taboolaBidAdapter.js

```diff
@@ -233,7 +233,7 @@ function getBid(requestId, currency, bidResponse) {
 
   return {
     requestId,
-    ttl: 60,
+    ttl,
```

Comment:
do you want to fallback to a default?

Reply:
just how I updated it as written

Reply:
added check in case of fallback with null value to set default one.